src/layman/layer/geoserver/__init__.py

```diff
@@ -64,28 +65,38 @@ def publish_layer_from_db(username, layername, description, title):
         title
     ]
     keywords = list(set(keywords))
+    feature_type_def = {
+        "name": layername,
+        "title": title,
+        "abstract": description,
+        "keywords": {
+            "string": keywords
+        },
+        "srs": "EPSG:3857",
+        "projectionPolicy": "FORCE_DECLARED",
+        "enabled": True,
+        "store": {
+            "@class": "dataStore",
+            "name": username + ":postgresql",
+        },
+    }
+    db_bbox = db_source.get_bbox(username, layername)
+    if db_bbox is None:
+        # world
+        native_bbox = {
+            "minx": -20026376.39,
+            "miny": -20048966.10,
+            "maxx": 20026376.39,
+            "maxy": 20048966.10,
+            "crs": "EPSG:3857",
+        }
+        feature_type_def['nativeBoundingBox'] = native_bbox
     r = requests.post(
         urljoin(settings.LAYMAN_GS_REST_WORKSPACES,
                 username + '/datastores/postgresql/featuretypes/'),
-        data=json.dumps(
-            {
-                "featureType": {
-                    "name": layername,
-                    "title": title,
-                    "abstract": description,
-                    "keywords": {
-                        "string": keywords
-                    },
-                    "srs": "EPSG:3857",
-                    "projectionPolicy": "FORCE_DECLARED",
-                    "enabled": True,
-                    "store": {
-                        "@class": "dataStore",
-                        "name": username + ":postgresql",
-                    },
-                }
-            }
-        ),
+        data=json.dumps({
```

Comment:
Jaké to má výhody proti nastavaní recalculate při následných změnách?

Reply:
Řešil jsem jen situaci při POST/PATCH Layer se zadaným parametrem file. Je fakt, že by si to zasloužilo víc testů i pro PATCH např. pouze title (zda zůstane bbox zachován). Přidám.

Tak jako tak není cílem tohoto PR automatická aktualizace bboxu při WFS-T.